scripts/build_ngx_pagespeed.sh

```diff
@@ -457,17 +469,22 @@ Not deleting $directory; name is suspiciously short.  Something is wrong."
     run tar -xzf "$nginx_fname" --directory "$BUILDDIR"
     "$DRYRUN" || cd "$nginx_dir"
 
+    configure=("./configure" "${configure_args[@]}")
     echo "About to build nginx.  Do you have any additional ./configure"
     echo "arguments you would like to set?  For example, if you would like"
     echo "to build nginx with https support give --with-http_ssl_module"
     echo "If you don't have any, just press enter."
     read -p "> " additional_configure_args
-
-    configure="./configure $configure_args $additional_configure_args"
+    if [ -n "$additional_configure_args" ]; then
+      # Split additional_configure_args respecting any internal quotation.
+      # Otherwise things like --with-cc-opt='-foo -bar' won't work.
+      eval additional_configure_args=($additional_configure_args)
+      configure=("${configure[@]}" "${additional_configure_args[@]}")
+    fi
     echo "About to configure nginx with:"
-    echo "  $configure"
+    echo "   $(quote_arguments "${configure[@]}")"
```

Comment:
I guess you added quotes because you want it to render --foo="--bar --baz" correctly, but this is going to look weird if your args include single quotes:
'--foo=--bar='baz''